wiki/Manual.md

```diff
@@ -6339,43 +6337,48 @@ Here is an outline of exception inheritance hierarchy: https://github.com/ccxt/c
 |   |
 |   +---+ NotSupported
 |
-+---+ NetworkError (recoverable)
-    |
-    +---+ InvalidNonce
-    |
-    +---+ RequestTimeout
-    |
-    +---+ ExchangeNotAvailable
-    |   |
-    |   +---+ OnMaintenance
+|
++---+ OperationFailed (recoverable)
     |
-    +---+ DDoSProtection
+    +---+ NetworkError (recoverable)
         |
-        +---+ RateLimitExceeded
+        +---+ InvalidNonce
+        |
+        +---+ RequestTimeout
+        |
+        +---+ ExchangeNotAvailable
+        |   |
+        |   +---+ OnMaintenance
+        |
+        +---+ DDoSProtection
+            |
+            +---+ RateLimitExceeded
 ```
 
 The `BaseError` class is a generic root error class for all sorts of errors, including accessibility and request/response mismatch. If you don't need to catch any specific subclass of exceptions, you can just use `BaseError`, where all exception types are being caught.
 
-From `BaseError` there are derived two different families of the error hierarchy:
-
-- `NetworkError`
-- `ExchangeError`
+From `BaseError` derives two different families of errors: `OperationFailed` and `ExchangeError` (they also have their specific sub-types, as explained below).
 
-They also have more specific sub-type exceptions, as explained below.
+### OperationFailed
+<a name="NetworkError" id="NetworkError"></a>
 
-### NetworkError
+An `OperationFailed` happens when user sends **correctly constructed & valid request** to exchange, but an non-deterministic problem occurred. In suchcases, you might consider retrying after some time.
+- maintenance ongoing
+- internet/network connectivitiy issues
```

Comment:
```suggestion
- rate limits exceeded
- internet/network connectivitiy issues
```

shouldnt we include RL note?

Reply:
I don't think so we have a specific exception for that, I think it might confuse the users

Reply:
`RateLimitExceeded` is that.. ?

Reply:
so we should list them under "temporary fails". it's the place where it belongs. maybe in opposite, missing that from the list might confuse users

Reply:
The thing is, we have a more specific exception to map the RL error (NetworkError -> DDOSPRotection -> RlExcedeed), we're already explaining the broader idea behind `OperatioNFailed` I think that is enough, the simpler the better

Reply:
the problem is that we dont have correct mapping (RL shoud not be under DDOS) so missing "rate limit issues" might confuse users into thinking as if "rate limits issues" is not listed under OpFailed. but sure, as you consider 👍

Reply:
now, as we moved our RL and made it sibling, we can place in that list. not?